static/js/views/installed-addon.js

```diff
@@ -249,6 +254,55 @@ class InstalledAddon {
         button.disabled = false;
       });
   }
+
+  /**
+   * Handle a click on the license button.
+   */
+  handleLicense(e) {
+    if (e.target.getAttribute('data-id')) {
+      API.getAddonsInfo().then((data) => {
+        const licenseUrl = `
```

Comment:
This is breaking the link. You're inserting space into it.

Reply:
If I remember correctly, if I don't immediately wrap like this, the linter give an error.

Reply:
Ah, it's the ` that was the issue.